### ISSUE TITLE
this repo is redundant

### ISSUE THREAD

**Issue by john-light** [onenameio/support#6]
Since Onename has support.onename.com, this repo probably doesn't need to exist (unless it is used as the main source of info that then gets copied over to Zendesk).

**Comment by Blake-at-Onename** [comment]
Resources on the help center are manually maintained & independent of this repo.  Maybe post in repo link to Onename Support?

**Comment by john-light** [comment]
I don't think we need to have a GitHub repo that just links to Zendesk. People who need help with code on GitHub can file an issue in the appropriate repo, and people who need help with the Onename product can just click the Support link on the website.
